### PR TITLE
TSPS-226 prechunk beagle inputs

### PR DESCRIPTION
**Pull request by jsotobroad**
### Description

_Give your PR a **concise** yet **descriptive** title._
_Please explain the changes you made here._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any issues fixed, addressed, or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
_You can delete these instructions once you have written your PR description._

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+        String interval_list_suffix = ".interval_list"
+        String bref3_suffix = ".bref3"
+    }
+
+    call tasks.CountSamples {
+        input:
+            vcf = multi_sample_vcf,
+    }
+
+    Float chunkLengthFloat = chunkLength
```
Comment (mmorgantaylor) [inline pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl:40]: looks like this isn't used anywhere?

File: tasks/broad/ImputationTasks.wdl [hunk 2]
```diff
@@ -1052,3 +1057,107 @@ task SplitMultiSampleVcf {
...
+    docker: gatk_docker
+    disks: "local-disk ${disk_size_gb} HDD"
+    memory: "${memory_mb} MiB"
+    cpu: cpu
+  }
+  output {
+    Array[File] generate_chunk_vcfs = glob("generate_chunk/*.vcf.gz")
+    Array[File] generate_chunk_vcf_indices = glob("generate_chunk/*.vcf.gz.tbi")
+    Array[File] subset_vcfs = glob("subset_vcf/*.vcf.gz")
+    Array[File] subset_vcf_indices = glob("subset_vcf/*.vcf.gz.tbi")
```
[the remark this answers is not in the document]
Comment (jsotobroad) [reply tasks/broad/ImputationTasks.wdl:1159]: yup will do. can remove it from the original subsetvcf task too

File: pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+
+            call tasks.CheckChunksBeagle {
+                input:
+                    var_in_original = CountVariantsInChunksBeagle.var_in_original,
+                    var_also_in_reference = CountVariantsInChunksBeagle.var_also_in_reference
+            }
+
+            call tasks.SetIDs as SetIdsVcfToImpute {
+                input:
+                    vcf = PreChunkVcf.subset_vcfs[i],
```
[the remark this answers is not in the document]
Comment (jsotobroad) [reply pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl:103]: SetIDs is a different docker so id rather not mess with that stuff in this PR yet.  I think that can be a further optimization

File: pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl (hunk 1 above)
Comment (jsotobroad) [reply pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl:40]: yup!

**Pull request merged**